### PR TITLE
Fix / update translations

### PR DESCRIPTION
**Pull request by arnaud-morvan**
TODO: Add tests in scaffold

### CODE REVIEW

File: c2cgeoform/templates/navigation.jinja2 [hunk 1]
```diff
@@ -15,7 +15,7 @@
       <ul class="nav navbar-nav">
         {% for table in tables %}
         <li role="presentation" class="{{'active' if request.matchdict and request.matchdict.get('table') == table['key'] else ''}}">
-          <a href="{{request.route_url('c2cgeoform_index', table=table['key'])}}">{{table['plural']}}</a>
+          <a href="{{request.route_url('c2cgeoform_index', table=table['key'])}}">{{_(table['plural'])}}</a>
```
Comment (sbrunner) [inline c2cgeoform/templates/navigation.jinja2]: plural is not already localized?

File: c2cgeoform/scaffolds/c2cgeoform/+package+/models/c2cgeoform_demo.py [hunk 2]
```diff
@@ -24,11 +24,9 @@
 from c2cgeoform.ext import colander_ext, deform_ext
 from c2cgeoform.models import FileData
 
+from .. import _
```
Comment (cmangeat) [inline c2cgeoform/scaffolds/c2cgeoform/+package+/models/c2cgeoform_demo.py]: wonder whether it make sense being be more explicit.

Comment (sbrunner) [reply c2cgeoform/scaffolds/c2cgeoform/+package+/models/c2cgeoform_demo.py]: like `from {{package}} import _`?
+1 for me :-)

Comment (arnaud-morvan) [reply c2cgeoform/scaffolds/c2cgeoform/+package+/models/c2cgeoform_demo.py]: Yes, I would prefer ``from {{package}} import _`` but this is not a template.

File: c2cgeoform/templates/navigation.jinja2 (hunk 1 above)
Comment (arnaud-morvan) [reply c2cgeoform/templates/navigation.jinja2]: it is already a TranslationString object, but for the moment I do not really understand why this is not translated without _ call.

File: c2cgeoform/scaffolds/c2cgeoform/+package+/models/c2cgeoform_demo.py (hunk 2 above)
Comment (sbrunner) [reply c2cgeoform/scaffolds/c2cgeoform/+package+/models/c2cgeoform_demo.py]: Then rename it ...

**Pull request merged**